mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -568,6 +571,53 @@ public void testIcebergAndHmsTableProperties() throws TException, InterruptedExc
     }
   }
 
+  @Test
+  public void testDropTableWithAppendedData() throws IOException {
+    TableIdentifier identifier = TableIdentifier.of("default", "customers");
+
+    shell.executeStatement(String.format("CREATE EXTERNAL TABLE %s STORED BY '%s' %s" +
```

Comment:
Would `testTables.createTable` be usable here?

Reply:
Yes, good idea.